packages/builders/src/jest/jest.builder.ts

```diff
@@ -16,37 +17,66 @@ try {
 const { runCLI } = require('jest');
 
 export interface JestBuilderOptions {
+  codeCoverage?: boolean;
   jestConfig: string;
+  testDirectory?: string;
+  testFile?: string;
+  setupFile?: string;
   tsConfig: string;
-  watch: boolean;
-  bail?: boolean;
+  bail?: boolean | number;
   ci?: boolean;
-  codeCoverage?: boolean;
-  onlyChanged?: boolean;
+  color?: boolean;
+  json?: boolean;
   maxWorkers?: number;
+  onlyChanged?: boolean;
+  outputFile?: string;
   passWithNoTests?: boolean;
   runInBand?: boolean;
-  setupFile?: string;
   silent?: boolean;
-  updateSnapshot?: boolean;
   testNamePattern?: string;
+  updateSnapshot?: boolean;
+  useStderr?: boolean;
```

Comment:
Can you please revert changes not relevant to `--test-file`?

Reply:
I was initially wrong about the minimum scope of changes required for integration.
All the added properties are used by vscode-jest in different scenarios (some - in debug mode, some - in the background).